proto/external_storage.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+// purposely leaving out delete. that is up to individual external
+// storage server operators to do on their end
+
+service ProxyStorage {
+
+    rpc GetModule (ModuleMetadata) returns (ModuleData) {}
```

Comment:
Not an expert in proto. But does this mean `@info` or `@mod` will download `zip_file` too?

Reply:
Yes, that matched the interface, at the time (before #406 landed).  
This service definition would need to be changed to match the storage interface.

Reply:
I think that Athens never downloads module@version metadata without also downloading the zip file. That said, does it make more sense to split this int `GetModuleZip` and `GetModuleMetadata`?